Specs/e2e/sandcastle.spec.js

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from "./test.js";
+import { globbySync } from "globby";
+
+const gallery = globbySync("Apps/Sandcastle/gallery/*.html");
+
+for (const example of gallery) {
+  test(`${example} renders`, async ({ page }) => {
+    test.setTimeout(100000);
+
+    await page.goto(example);
+
+    await page.waitForLoadState("networkidle");
+
+    await page.evaluate(() => window.__clock.tick(1000));
+    await page.evaluate(() => window.__clock.tick(1000));
+    await page.evaluate(() => window.__clock.tick(1000));
+    await page.evaluate(() => window.__clock.tick(1000));
```

Comment:
These `page.evaluate` calls gave me a confusing error in one test run:

```bash
  1) [chromium] › sandcastle.spec.js:7:3 › Apps/Sandcastle/gallery/Particle System Weather.html renders

    Error: page.evaluate: TypeError: Cannot read properties of undefined (reading 'tick')

        at eval (eval at evaluate (:197:30), <anonymous>:1:22)
        at UtilityScript.evaluate (<anonymous>:199:17)
        at UtilityScript.<anonymous> (<anonymous>:1:44)
        at eval (/home/jhembd/CesiumGS/cesium/eval at evaluate (:197:30), <anonymous>:1:22)
        at UtilityScript.evaluate (/home/jhembd/CesiumGS/cesium/<anonymous>:199:17)
        at UtilityScript.<anonymous> (/home/jhembd/CesiumGS/cesium/<anonymous>:1:44)
        at /home/jhembd/CesiumGS/cesium/Specs/e2e/sandcastle.spec.js:14:16
```

I could avoid it by using optional chaining, to handle the case `window.__clock === undefined`. 

```javascript
await page.evaluate(() => window.__clock?.tick(1000));
```

This gave me a more meaningful error message.

Reply:
This means that the mock clock is not getting installed in time, which is a bigger issue... I'll take a look.